php/php-mailparse/Portfile

```diff
@@ -45,6 +45,11 @@ if {${name} ne ${subport}} {
     post-destroot {
         set docdir ${prefix}/share/doc/${subport}
         xinstall -d ${destroot}${docdir}
-        xinstall -m 644 -W ${worksrcpath} CREDITS README ${destroot}${docdir}
+        xinstall -m 644 -W ${worksrcpath} CREDITS ${destroot}${docdir}
+        if {[vercmp ${version} 3.0.3] >= 0} {
```

Comment:
The README has been renamed. I wasn't sure if this was the best way to resolve this for different versions. I've tested this for `php73-mailparse` and `php56-mailparse` to make sure that both variations still work.